### PR TITLE
Fix: Initialization error, install vite

### PR DESCRIPTION
**Pull request by jessmitch42**
Reported in [issue #8](https://github.com/daily-demos/svelte-call-object/issues/8).

The app wouldn't load and needed some recent Svelte fixes to resolve, including install Vite. Fix mentioned [here](https://www.youtube.com/watch?v=DSaRNhPpwMM).

### CODE REVIEW

File: src/lib/call/Controls.svelte [hunk 1]
```diff
@@ -56,12 +62,6 @@
 		document?.body?.classList?.remove('in-call');
 		goto(`/`);
 	};
-
-	onMount(() => {
-		if (!callObject) return;
-		camOn = callObject.localVideo();
-		micOn = callObject.localAudio();
-	});
```
Comment (jessmitch42) [inline src/lib/call/Controls.svelte:64]: there were two `onMount`s, whoops!

File: src/routes/room/index.json.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export async function post() {
+export async function POST() {
```
Comment (jessmitch42) [inline src/routes/room/index.json.js:1]: required change to make that uppercase in new sveltekit version

**Comment by jessmitch42** [comment]
@lazeratops I'm going to merge this since `main` is broken. If you could review when you have a second, I can include any suggestions you have in a follow-up PR 🙏

**Review by lazeratops — COMMENTED** [review]
Belated lgtm!

**Pull request merged**